doc/introduction.rst

```diff
@@ -0,0 +1,50 @@
+.. _Introduction to XBlocks:
+
+#############################
+Introduction to XBlocks
+#############################
+
+You use the XBlocks APIs to build components that course teams can use to
+create independent course components that work seamlessly with other components
+in an online course.
+
+For example, you can build XBlocks to represent individual problems or pieces
+of text or HTML content. Furthermore, like Legos, XBlocks are composable; you
+can build XBlocks to represent larger structures such as lessons, sections, and
+entire courses.
+
+A primary advantage to XBlocks is that they are deployable. The code you write
```

Comment:
I don't understand what deployable means here, in contrast to any other code inside edx-platform.  "Sharable"?